_data/users.yml

```diff
@@ -0,0 +1,65 @@
+- name: Ampool
+  anchor: ampool
+  logo: /assets/images/logos/ampool.png
+  testimonial: |
+    Ampool is powered by Trino to provide SQL query federation across 200+ data
+    sources. Combined with Ampool’s high-performance in-memory distributed
+    caching engine, a turbocharged Trino query engine can speedup BI & ad-hoc
+    analytical workloads by an order of magnitude. Ampool is built from the
+    ground up to operate seamlessly in hybrid and multi-cloud environments.
+  urltext: Visit the Ampool website for more information!
+  url: https://www.ampool-inc.com/
+- name: Gett
+  anchor: gett
+  logo: /assets/images/logos/gett.png
+  testimonial: |
+    As the leading ground travel platform for businesses, Gett uses Trino as the
+    core SQL engine for all data pipelines. Extracting data from heterogeneous
+    sources, transforming it with the rich analytical functions and loading
+    results to the visualisation tools to get wiser business decisions.
+  urltext:
+  url: https://gett.com/intl/
+- name: Jampp
+  anchor: jampp
+  logo: /assets/images/logos/jampp.png
+  testimonial: |
+    Jampp employs Trino as the main interface between its Data Warehouse and the
+    rest of the company, using it to feed their predictive algorithms as well as
+    performing analytic queries, ETLs and monitoring data quality.
+  urltext:
+  url: https://jampp.com/
+- name: LINE
+  anchor: line
+  logo: /assets/images/logos/line.png
+  testimonial: |
+    Trino is a core piece of LINE’s BI tools and interactive analysis workflow.
+    It is being actively used in areas such as KPI and system metrics
+    monitoring, as well as data analytics for large datasets. Trino is the
+    preferred tool for our ML engineers and data scientists to better understand
+    the data on our datalake through fast, interactive queries.
+  urltext:
+  url: http://line.me
```

Comment:
Can we use the below url? The current url is for LINE app (one of our services). 
https://linecorp.com/en

Reply:
Done